source/upgrade/important-upgrade-notes.rst

```diff
@@ -21,6 +21,10 @@ Important Upgrade Notes
 +----------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | If you’re upgrading from a version earlier than... | Then...                                                                                                                                                          |
 +====================================================+==================================================================================================================================================================+
+| v6.2                                               | Channel results in the channel switcher will include channels from other teams as well as private channels. For users using Bleve or Elasticsearch for           |
+|                                                    | autocomplete/search will have to reindex their data to get the new results. Since this can take a long time, it is suggested to disable autocomplete and run     |
```

Comment:
```suggestion
|                                                    | autocomplete/search will have to reindex their data to get the new results. Since this can take a long time, we suggest disabling autocomplete and running     |
```

Reply:
My proposed updates may break the table from displaying properly -- it was challenging to manage the table syntax with these edits.